packages/types/src/definitions/core/ProjectData.type.ts

```diff
@@ -5,4 +5,5 @@ export type ProjectData = {
   publicInfo: string;
   backstageUrl: string;
   backstageInfo: string;
+  projectLogo?: string;
```

Comment:
question: we seem to have some varying approaches here. We define projectLogo as `string | undefined`, but use `empty string` to signify no logo

Would it be better to have this field be an union `string | null` so that the value is more explicit?

Reply:
I changed it to `string | null`, but would this break existing projects that do not have that property on the json?

Reply:
thank you for bringing this up. added a test to cover this case